parser/parser.go

```diff
@@ -30,45 +29,25 @@ type Package struct {
 }
 
 // ParsePackage parses the package in the given directory and returns it.
-func ParsePackage(directory, skipPrefix, skipSuffix string) (*Package, error) {
```

Comment:
Because I don't know how to golang.org/x/tools/go/loader skip certain files. I'm also quite unconvinced that it's necessary; as far as I can tell, all this does is prevent recursive invocation of jsonenums on its own output, but as each type you want to json-enumerate has to be explicitly mentioned in the go generate arguments, you couldn't accidentally recurse, only intentionally (which should be fine). What am I missing?